packages/client/components/EmailPasswordAuthForm.tsx

```diff
@@ -172,12 +173,13 @@ const EmailPasswordAuthForm = forwardRef((props: Props, ref: any) => {
       return false
     }
     submitMutation()
-    const {token, error} = await getTokenFromSSO(url)
+    const {token, error, ga4Args} = await getTokenFromSSO(url)
     if (!token) {
       onError(new Error(error || 'Error logging in'))
       return true
     }
     atmosphere.setAuthToken(token)
+    emitGA4SignUpEvent(ga4Args!)
```

Comment:
+1 I don't particular like this, but I see because the token is valid, these are as well. Could you please add a comment stating this?